src/samples/EducationStart/ClaimList.tsx

```diff
@@ -83,6 +83,10 @@ export default function ClaimsList(props) {
 
     const target = `${PCore.getConstants().APP.APP}/${container}`;
 
+    sessionStorage.setItem('isComingFromPortal', 'true');
+    sessionStorage.setItem('isEditMode', 'true');
```

Comment:
As we are using the session storage here, I think that this should be covered by a unit test. Something like 
`export const isSessionStorageAvailable = () => {
  try {
    const testKey = '__storage_test__';
    sessionStorage.setItem(testKey, testKey);
    sessionStorage.removeItem(testKey);
    return true;
  } catch (e) {
    return false;
  }
};`
Then a Unit test that would check if the storage is available?
`
//ClaimList.Test.tsx
import { isSessionStorageAvailable } from './storageUtils';
describe('isSessionStorageAvailable', () => {
  it('should return true if sessionStorage is available', () => {
    expect(isSessionStorageAvailable()).toBe(true);
  });
  it('should return false if sessionStorage is not available', () => {
    const originalSessionStorage = window.sessionStorage;
    Object.defineProperty(window, 'sessionStorage', {
      value: null,
      writable: true,
    });
    expect(isSessionStorageAvailable()).toBe(false);

    // Restore original sessionStoragewindow.sessionStorage = originalSessionStorage;
  });
  it('should return false if sessionStorage throws an error', () => {
    const originalSetItem = window.sessionStorage.setItem;
    window.sessionStorage.setItem = jest.fn(() => {
      throw new Error('Storage error');
    });

    expect(isSessionStorageAvailable()).toBe(false);

    // Restore original setItemwindow.sessionStorage.setItem = originalSetItem;
  });
});`